R/eml.R

```diff
@@ -29,6 +29,61 @@ pid_to_eml_other_entity <- function(mn, pids) {
   sysmeta_to_eml_other_entity(sysmeta)
 }
 
+#' Create an EML dataTable object for a given PID. 
+#' This function generates an attributeList and physical and constructs a dataTable.
+#'
+#' @param mn (MNode) Member Node where the PID is associated with an object.
+#' @param pid (character) The PID of the object to create the dataTable for.
+#' @param attributes (data.frame) Data frame of attributes.
```

Comment:
I don't know about you, but I have seen a lot of packages say the argument should be a `data.frame` but stops short of telling me how to structure it. If you can assume the person is an `EML` package expert, maybe they know what to do here. But I think it might be more clear if you add a sentence here (and for `@param factor) like 

> Follows the convention in `\link[EML]{set_attributes}`

which will both (1) tell them which package and function has the info they need and (2) helpfully provide a link they can click on to go to that help page.

What do you think?

Reply:
Totally agree. Will add some documentation to clarify the structure.